service/src/integrationTest/resources/application.yaml

```diff
@@ -1,5 +1,6 @@
 spring:
   main:
+    allow-bean-definition-overriding: true
```

Comment:
yeah, it is necessary. Some beans of those APIs are registered multiple time. The alternative way to do it is adding excludingFilters to ComponentScan for each duplicated beans, e.g. RetryConfig.

But yeah, I think it is not safe to turn bean overriding on, it may potentially leads to future bugs if other developers doesn't aware this is toggled on.